content/en/docs/configuration/acme/dns01/cloudflare.md

```diff
@@ -5,6 +5,69 @@ weight: 30
 type: "docs"
 ---
 
+To use Cloudflare, you may use one of two types of tokens. **API Tokens** allow application-scoped keys bound to specific zones and permissions, while **API Keys** are globally-scoped keys that carry the same permissions as your account.
```

Comment:
I think other areas of the docs use "CloudFlare", hence why "Cloudflare" is being flagged for incorrect spelling!

Reply:
Alright, I (hopefully) just fixed the two spelling errors it caught.